code/modules/mob/living/carbon/human/life/handle_fire.dm

```diff
@@ -8,4 +8,41 @@
 
 	var/thermal_protection = get_heat_protection(get_heat_protection_flags(30000)) //If you don't have fire suit level protection, you get a temperature increase
 	if((1 - thermal_protection) > 0.0001 && bodytemperature < T0C+100) //MATHEMATICAL HELPERS FOR FUCKS SAKES
-		bodytemperature = min(bodytemperature + BODYTEMP_HEATING_MAX,T0C+100)
\ No newline at end of file
+		bodytemperature = min(bodytemperature + BODYTEMP_HEATING_MAX,T0C+100)
+		var/head_exposure = 1
+		var/chest_exposure = 1
+		var/groin_exposure = 1
+		var/legs_exposure = 1
+		var/arms_exposure = 1
+
+		//Get heat transfer coefficients for clothing.
+
+		for(var/obj/item/clothing/C in src)
+			if(is_holding_item(C))
+				continue
+
+			if( C.max_heat_protection_temperature >= bodytemperature)
+				if(!is_slot_hidden(C.body_parts_covered,FULL_HEAD))
+					head_exposure = 0
+				if(!is_slot_hidden(C.body_parts_covered,UPPER_TORSO))
+					chest_exposure = 0
+				if(!is_slot_hidden(C.body_parts_covered,LOWER_TORSO))
+					groin_exposure = 0
+				if(!is_slot_hidden(C.body_parts_covered,LEGS))
+					legs_exposure = 0
+				if(!is_slot_hidden(C.body_parts_covered,ARMS))
+					arms_exposure = 0
+
+		apply_damage(2.5*head_exposure, BURN, LIMB_HEAD, 0, 0, used_weapon = "Fire")
```

Comment:
same mods appear in 2 places? probably a define moment

Reply:
I don't have a clue how these initial numbers were created or what they were based on. Any suggestions on what to use as the defines for these?

Reply:
Just explain what it is with the define name. For example
BURN_MOD_HEAD
BURN_MOD_CHEST
etc

Reply:
Fair enough, will do